docs/Configuration.md

```diff
@@ -20,20 +20,23 @@ You can add these configurations into spark-defaults.conf to enable or disable t
 | spark.plugins                                           | To load Gluten's components by Spark's plug-in loader                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | com.intel.oap.GlutenPlugin                           |
 | spark.shuffle.manager                                   | To turn on Gluten Columnar Shuffle Plugin                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | org.apache.spark.shuffle.sort.ColumnarShuffleManager |
 | spark.gluten.enabled                                    | Enable Gluten, default is true. Just an experimental property. Recommend to enable/disable Gluten through the setting for `spark.plugins`.                                                                                                                                                                                                                                                                                                                                                                                                | true                                                 |
+| spark.gluten.sql.columnar.maxBatchSize                  | Number of rows to be processed in each batch. Default value is 4096.                                                                                                                                                                                                                                                                                                                                                                                                | true                                                 |
```

Comment:
The default value should be 4096, not true or false.

Reply:
done